src/features/AnnouncementModal/Announcements.ts

```diff
@@ -0,0 +1,26 @@
+import { IsomerThumbsUp } from "assets"
+import { IsomerWaitingLine } from "assets/images/IsomerWaitingLine"
+import { Announcements } from "types/announcements"
+
+// TODO: Add link
+export const ANNOUNCEMENTS: Announcements[] = [
+  {
+    link: "",
+    announcements: [
+      {
+        title: "Control who can edit your website",
+        description:
+          "Isomer now lets you manage your site’s Collaborators, which includes Admins and Contributors. Only Admins can add or remove these.",
+        image: IsomerWaitingLine,
+        tags: ["New Feature"],
+      },
+      {
```

Comment:
Are we supporting like multiple announcements? Or only 1 announcement at a time?

Reply:
supposedly multiple which is why it's like this

Reply:
okay cool